neuralmagic/benchmarks/run_benchmark_serving.py

```diff
@@ -0,0 +1,71 @@
+import subprocess
+import time
+import socket
+
+from typing import NamedTuple
+from pathlib import Path
+
+from neuralmagic.tools.call_cmd import call_cmd
+from common import download_datasets, script_args_to_cla
+
+BENCH_SERVER_HOST = "localhost"
+BENCH_SERVER_PORT = 9000
+
+def get_this_script_dir() -> Path:
+    return Path(__file__).parent.resolve()
+
+def is_server_running(host: str, port: int, timeout=20) -> bool:
```

Comment:
TODO : sometimes it takes way too longer than 20s to start a server. Find a reasonable workaround for this.

Reply:
So, it takes time for model download -  I added a separate download step and bumped up the timeout to a minute . This should be fine.